package.json

```diff
@@ -8,6 +8,7 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "cross-env": "^7.0.3",
```

Comment:
Should this be a dev dependency?

Reply:
Oooh, yes... not that the distinction really matters, the build actually walks the graph via code, and doesn't refer to `package.json` at all. But for the sake of my ocd i will move.

Reply:
Done